pytorch_pfn_extras/utils/checkpoint.py

```diff
@@ -0,0 +1,42 @@
+import torch.utils.checkpoint
+
+
+class _CheckpointFunction(torch.utils.checkpoint.CheckpointFunction):
+    """Checkpoint a model or part of the model with BN support.
+
+    Refer to https://pytorch.org/docs/stable/checkpoint.html
+    for detailed information.
+    When using checkpointing in model using BatchNormalization, the
+    momentum is updated twice, while we only need one update to ensure
+    correctness.
+    Using `ppe.utils.checkpointing.checkpoint` as a drop-in replacement
+    can help to lead with incorrect values in the backward pass generated
```

Comment:
Actually, this PR doesn't change the values in the backward pass. It just makes sure that the persistent avg mean/var parameters are updated correctly (which will be used during test time). Gradients will be the same with and without this PR.

Reply:
Perhaps the doc should be something like
```Using `ppe.utils.checkpointing.checkpoint` as a drop-in replacement
    can help deal with incorrect values in the BatchNormalization persistent parameters.
```